torchci/pages/minihud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -305,8 +473,39 @@ function MiniHud({ params }: { params: HudParams }) {
 
   return (
     <>
-      {shaGrid.map((row: RowData) => (
-        <CommitSummary row={row} key={row.sha} />
+      <div>
+        <input
+          type="checkbox"
+          id={"showDurationInfo"}
```

Comment:
I don't think you need to name this showDurationInfo. Could be kinda confusing if people are searching for this variable.

Reply:
changed to a different name, but left the id and htmlFor fields, i think it lets you click on the label instead of the actual box?